nette/notejam/app/presenters/HomepagePresenter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Presenters;
+
+use App\Components\Notes\INotesFactory;
+use App\Helpers\OrderHelper;
+use App\Model\NoteManager;
+use Nette;
+
+
+class HomepagePresenter extends SecuredBasePresenter
+{
+
+	/** @var NoteManager @inject */
+	public $noteManager;
+
+	/** @var INotesFactory @inject */
+	public $notesFactory;
+
+	/** @var object[] */
+	protected $notes;
+
+
+	/********************* Controls ***********************************************************************************/
```

Comment:
`/********************* translator ****************d*g**/`

What means `d` and `g`? :)